packages/core/src/typegenTypes.ts

```diff
@@ -123,9 +123,17 @@ export type TypegenConstraint = TypegenEnabled | TypegenDisabled;
  */
 export type AreAllImplementationsAssumedToBeProvided<
   _TResolvedTypesMeta,
-  _TMissingImplementations
+  _TMissingImplementations = never
```

Comment:
There is a q if we even need to do this - but I think we should. We left `AreAllImplementationsAssumedToBeProvided` for compat reasons after all in https://github.com/statelyai/xstate/pull/4600 ... we just didn't account for its default type param (integration packages were only supplying one type arg) and we didn't do the same for `MissingImplementationsError`. 

The goal **was** to preserve compatibility and avoid issues and that fixes it. Since it's easy to do this, I think it's worth considering.

Reply:
I think this is fine